src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir-book", sans-serif;
+  font-size: 16px;
+}
+
+.search {
+  display: flex;
+  margin: 20px 0;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search--big {
+  height: 70px;
+}
+
+.search--small {
+  height: 42px;
+}
+
+.label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: 0;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.input--big {
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input--small {
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.input--big::placeholder {
+  color: inherit;
+  font-weight: 300;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.input--small::placeholder {
+  color: inherit;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+  font-weight: 600;
```

Comment:
font-weight: 600; doesn't change now on demo
apply outline: none; for normal state of element

Reply:
changed